src/test/aks/setup-env-variables-template.sh

```diff
@@ -1,9 +1,9 @@
-export WLS_AKS_REPO_REF="278a527cb7d1eb44a5a65ccc02b6837fe42967db" # oracle/weblogic-azure reference
-export RESOURCE_GROUP_NAME="abc1110rg" # customize this
-export STORAGE_ACCOUNT_NAME="stgwlsaks$(date +%s)" # storage account name
-export DB_SERVER_NAME="wlsdb$(date +%s)" # PostgreSQL server name
-export DB_PASSWORD="Secret123456" # PostgreSQL database password
-export MY_ORACLE_SSO_USER="user@contoso.com" # replace with your Oracle Account user name.
-export MY_ORACLE_SSO_PASSWORD="Secret123456" # replace with your Oracle Account password.
-export MY_WEBLOGIC_ADMIN_USER_NAME="weblogic" # weblogic admin user name
-export MY_WEBLOGIC_ADMIN_PASSWORD="Secret123456" # weblogic admin password
+export WLS_AKS_REPO_REF="2024-03-13-2-Q1" # oracle/weblogic-azure reference

```

Comment:
Update the tag to `2024-03-21-1-Q1`.

Reply:
Resolved in https://github.com/Azure-Samples/cargotracker-wls-aks/pull/7/commits/635c0079fb9d5c31d681d07e6cc147bb446393c1